week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -26,7 +42,8 @@ The function must:
 */
 
 function remove(arr, index) {
-  return; // complete this statement
+  arr.splice(index, 1);
```

Comment:
So there is a mistake here. while your fix will pass, but it won't follow the condition (not change original array)
try to search splice destructive js 
and try to do it using slice (twice maybe)